docs/usage/configuration/config_documentation.md

```diff
@@ -1380,6 +1380,22 @@ rc_joins:
     burst_count: 12
 ```
 ---
+### `rc_joins_per_room`
+
+This option allows for ratelimiting joins to a room based on the number of recent
+joins (local or remote) to that room. It is intended to mitigate mass-join spam
+waves which target multiple homeservers.
+
+Sensible values for this option are provided by default; most server admins
```

Comment:
I think the defaults should be explicitly stated here, looking at the code looks like it is 1 per second and 10 burst count?

Reply:
The defaults are plucked out of thin air, tbh---as part of this review I'd like to solicit opinions on whether that's sensible. I can add them here though.